src/translations/fr.json

```diff
@@ -1018,5 +1018,6 @@
     "Create": "Créer",
     "ShuntCompensator": "Moyen de compensation statique",
     "AttachingAndSplittingLines": "Piquage et coupure",
-    "GenerationAndLoad": "Production et consommation"
+    "GenerationAndLoad": "Production et consommation",
+    "SpreadsheetFetchError": "Une erreur est survenue lors du chargement des ouvrages"
```

Comment:
```suggestion
    "SpreadsheetFetchError": "Une erreur est survenue lors du chargement des ouvrages du tableur"
```
Same for english

Reply:
english : An error occurred while fetching equipments in spreadsheet 
is it ok ?

Reply:
To be the most correct and coherent possible, I would write :

Une erreur est survenue lors du chargement d'ouvrages dans le tableur
An error occurred while fetching equipments in the spreadsheet

But it's fine